src/app/containers/NewStoryPromo/PromoOrganisms/TopStoriesPromo/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { arrayOf, shape } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import pathOr from 'ramda/src/pathOr';
+import { C_GREY_2 } from '#app/legacy/psammead-styles/src/colours';
+import Promo from '../../Promo';
+import { TopStoriesPromoWrapper } from './index.styles';
+import TopStoriesItem from './TopStoriesMolecules/TopStoriesItem';
+import TopStoriesList from './TopStoriesMolecules/TopStoriesList';
+
+const TopStoriesPromo = ({ content }) => {
+  const { script, service, dir, translations } = useContext(ServiceContext);
+  const title = pathOr('Top Stories', ['topStoriesTitle'], translations);
+  const hasSingleContent = content.length === 1;
+  const LABEL_ID = 'top-stories-heading';
+
+  return (
+    <Promo Wrapper={TopStoriesPromoWrapper}>
```

Comment:
Passing a component as a prop is something that can lead to complexity.  I think just doing this is probably better:

```
<Promo>
  <TopStoriesPromoWrapper>
    Blah
  </TopStoriesPromoWrapper>
</Promo>
```

Reply:
Yep, I finally understood how to apply dynamic styles to the Wrapper. So now this is fixed by using `className` in the Promo component.